src/components/Navbar.jsx

```diff
@@ -2,35 +2,49 @@ import { Link, Outlet } from 'react-router-dom';
 import { Navbar as Nav, NavbarBrand, NavbarContent, NavbarItem, Avatar } from '@nextui-org/react';
 import { PlusOutlined } from '@ant-design/icons';
 import ProfileAvatar from './ProfileAvatar';
+import Searchbar from './Searchbar.jsx';
 
 const Navbar = () => {
   return (
     <div>
-      <Nav className="bg-transparent py-2" position="static" height="3rem" shouldHideOnScroll>
-        <NavbarBrand>
-          <Link to="/">BRUIN DICTIONARY</Link>
-        </NavbarBrand>
-        <NavbarContent className="hidden sm:flex gap-4" justify="center">
-          <NavbarItem>
-            <Link to="/">HOME</Link>
-          </NavbarItem>
-        </NavbarContent>
-        <NavbarContent justify="end">
-          <NavbarItem>
-            <Link to="/add">
-              <Avatar color="primary" icon={<PlusOutlined className="text-base" />} size="sm" />
-            </Link>
-          </NavbarItem>
-          <NavbarItem>
-            <Link to="/login">
-              <ProfileAvatar />
-            </Link>
-          </NavbarItem>
-        </NavbarContent>
-      </Nav>
+      <div className="bg-gray-700 space-y-1 pb-4">
+        <Nav className="bg-gray-700" position="static" height="3rem" shouldHideOnScroll>
+          <NavbarBrand>
+            <NavLink to="/" className="font-bold text-lg">
+              BRUIN DICTIONARY
+            </NavLink>
+          </NavbarBrand>
+          <NavbarContent className="hidden sm:flex gap-4 justify-center">
```

Comment:
let's remove `justify-center` in className and add `justify="center"` attribute

Reply:
`BRUIN DICTIONARY` and `HOME` are basically the same now, but I was thinking add some extra features pages here.